text/3501-edition-2024.md

```diff
@@ -0,0 +1,88 @@
+- Feature Name: `edition_2024`
+- Start Date: 2023-09-25
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC is an amendment to [RFC 3085] to declare that the Rust Project intends to produce a new edition in 2024,
+to set up a [project group] to deliver the edition,
+and to establish a tentative cadence for future editions.
+
+[RFC 3085]: https://rust-lang.github.io/rfcs/3085-edition-2021.html
+[project group]: https://rust-lang.github.io/rfcs/2856-project-groups.html
+
+# Motivation
+[motivation]: #motivation
+
+So far, editions have been delivered on a three-year cadence, with the 2015, 2018, and 2021 editions.
+Many within the Rust project have been expecting this trend to continue.
+The three-year cadence seems to hit a reasonable balance of not too often, but still providing opportunities for potentially breaking changes to be introduced.
+
+By establishing a three-year cadence, this removes the ambiguity of what the expectations are around scheduling and releases of new editions.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## 2024 Edition
+
+The Rust Project intends to create a 2024 Edition.
+Currently, no specific changes are being announced in this RFC for this edition.
+Team members will be responsible for identifying changes they want to make and coordinate with the Edition project group.
+
+## Edition project group
+
+The Leadership Council will be responsible for forming a project group for coordinating each edition release.
```

Comment:
It seems like we might want to say where the project group lives in the project.. maybe that can be an unresolved question? For now, I suppose launching pad is the reasonable place. (It seems like we might expect it to be around regularly for ~1+ years so probably not quite ephemeral enough to skip this question).

Reply:
I was expecting it to be parented to the council as a whole. It could be part of the launching pad, but I'm not sure in what ways it matters?

Reply:
I think the main thing is that e.g. in processes like project director elections or council elections, we want to be clear about who is in under what bucket(s). I think the edition group probably isn't active ~1/3-2/3 of the time so it's maybe not as important that there's clear representation, but I think trying to keep to that pattern is a good idea as much as we can. Explicitly putting it under launching pad seems like an easy way to achieve that goal for now.

Reply:
Sure! It's a bit awkward right now since the launching pad isn't in the team database, but I think it should work for now. Also, it seems like the intent was to never have anything permanently in the launching pad, but this seems reasonable to figure out at some point later. The edition group is somewhat unique because it is a project-wide effort affecting many teams, so there isn't a clear way (to me) to organize it.